kansha/card_addons/members/models.py

```diff
@@ -5,4 +5,52 @@
 # This software is licensed under the BSD License, as described in
 # the file LICENSE.txt, which you should have received as part of
 # this distribution.
-#--
\ No newline at end of file
+#--
+
+from nagare import database
+from sqlalchemy import func
+from elixir import ManyToOne, ManyToMany, using_options
+
+from kansha.models import Entity
+from kansha.card.models import DataCard
+from kansha.user.models import DataUser
+from kansha.column.models import DataColumn
+
+
+class DataMembership(Entity):
+    using_options(tablename='user_cards__card_members')
+
+    user = ManyToOne(DataUser, primary_key=True)
+    card = ManyToOne(DataCard, primary_key=True)
+
+    @classmethod
+    def get_for_card(cls, card):
+        return cls.query.filter_by(card=card)
+
+    @staticmethod
+    def favorites_for(card):
+        query = database.session.query(DataUser)
```

Comment:
I prefer to keep the request simple for the moment, as all known boards in production have few members.

This part will be rewritten when we unify membership anyway.